src/js/Favorites.js

```diff
@@ -0,0 +1,208 @@
+import React, { useState, useEffect } from "react";
+import "../css/Favorites.css";
+
+function Favorites() {
+  const [profile, setProfile] = useState();
+  const [folder, setFolder] = useState(null);
+
+  useEffect(() => {
+    const fetchUserData = async () => {
+      try {
+        const response = await fetch(
+          "https://bootcamp-api.codeit.kr/api/sample/user"
+        );
+        const data = await response.json();
+        if (data) {
+          setProfile(data);
+        } else {
+          setProfile();
+        }
+      } catch (error) {
+        console.error("User Error:", error);
+        setProfile();
+      }
+    };
+
+    const fetchFolderData = async () => {
+      try {
+        const response = await fetch(
+          "https://bootcamp-api.codeit.kr/api/sample/folder"
+        );
+        const data = await response.json();
+        if (data) {
+          setFolder(data.folder);
+        } else {
+          setFolder();
+        }
+      } catch (error) {
+        console.error("Folder Error:", error);
+        setFolder();
+      }
+    };
+
+    fetchUserData();
+    fetchFolderData();
+  }, []);
+
+  function timeAgo(createdAt) {
```

Comment:
일반적으로 함수명은 동사+명사로 네이밍 하는 편이에요. 이미 잘 네이밍하신 다른 함수들처럼 동사+명사로 변경하면 쉽게 함수임을 인지할 수 있을것 같습니다!
```suggestion
  function getTimeAgo(createdAt) {
```